be/src/runtime/exec_env.cpp

```diff
@@ -153,9 +154,19 @@ Status ExecEnv::_init(const std::vector<StorePath>& store_paths) {
                             .set_max_queue_size(1000)
                             .set_idle_timeout(MonoDelta::FromMilliseconds(2000))
                             .build(&driver_dispatcher_thread_pool));
-    _driver_dispatcher = new pipeline::GlobalDriverDispatcher(std::move(driver_dispatcher_thread_pool));
+    _driver_dispatcher = new pipeline::GlobalDriverDispatcher(std::move(driver_dispatcher_thread_pool), false);
     _driver_dispatcher->initialize(max_thread_num);
 
+    std::unique_ptr<ThreadPool> wg_driver_dispatcher_thread_pool;
+    RETURN_IF_ERROR(ThreadPoolBuilder("pip_wg_dispatcher") // pipeline dispatcher
```

Comment:
Is it safe to destroy the object if `_init` return error halfway?

Reply:
`main()` calls `ExecEnv::init()`, but doesn't check its returned status. Let me update it.